core-data/resolvers.js

```diff
@@ -36,14 +37,17 @@ export async function* getMedia( state, id ) {
 }
 
 /**
- * Requests a post type element from the REST API.
+ * Requests a entity's record from the REST API.
  *
- * @param {Object} state State tree
- * @param {number} slug  Post Type slug
+ * @param {Object} state  State tree
+ * @param {string} kind   Entity kind.
+ * @param {string} name   Entity name.
+ * @param {number} key    Record's key
  */
-export async function* getPostType( state, slug ) {
-	const postType = await apiRequest( { path: `/wp/v2/types/${ slug }?context=edit` } );
-	yield receivePostTypes( postType );
+export async function* getEntityRecord( state, kind, name, key ) {
+	const entity = getEntity( kind, name );
```

Comment:
If we use `entityConfig` it means we need an `entity` too, but we don't have such a thing.

I think it's just `entity`. I've used `entityConfig` in the reducer to avoid variable shadowing.